extensions/smallrye-opentracing/runtime/src/main/java/io/quarkus/smallrye/opentracing/runtime/HystrixTracingConcurrencyInitializer.java

```diff
@@ -0,0 +1,24 @@
+package io.quarkus.smallrye.opentracing.runtime;
+
+import javax.enterprise.context.Dependent;
+import javax.enterprise.event.Observes;
+
+import org.jboss.logging.Logger;
```

Comment:
is jboss logging preferred logging API in quarkus?

Reply:
Yes.